sandpack-react/src/hooks/usePreviewProgress.ts

```diff
@@ -1,7 +1,7 @@
 import type { WorkerStatusUpdate } from "@codesandbox/nodebox";
```

Comment:
Would you mind renaming this hook to `useSandpackPreviewProgress` and exporting it on `hooks/index.ts`?

Reply:
of course